include/cuda/std/barrier

```diff
@@ -293,7 +293,7 @@ inline void __strided_memcpy(char * __destination, char const * __source, std::s
     }
 }
 
-#if __CUDA_ARCH__ >= 800
+#if __CUDA_MINIMUM_ARCH__ >= 800
```

Comment:
Fixed remaining __CUDA_ARCH__ dispatches.

Reply:
I'm not seeing any further changes in this file. Line 389, for one, isn't using nvtarget.

Reply:
Hmm... That's because barrier fixes are part of another patch set. I won't be pulling those ones into it. Instead I can reset this file to main to avoid complicating things further.